src/frontend/src/common/components/library/data_subview/index.tsx

```diff
@@ -68,41 +97,30 @@ const DataSubview: FunctionComponent<Props> = ({
     dispatch({ results: filteredData, searching: false });
   };
 
-  const tsvDataMap = (tableData: TableItem[]): [string[], string[][]] => {
-    const tsvData = tableData.map(entry => {
-      const values = headers.flatMap(header => {
-        if (typeof entry[header.key] === "object" && subheaders.hasOwnProperty(header.key)) {
-          const subEntry = entry[header.key] as Record<string, JSONPrimitive>
-          return subheaders[header.key].map(subheader => String(subEntry[subheader.key]))
-        }
-        return String(entry[header.key])
-      })
-      return values
-    })
-    const tsvHeaders = headers.flatMap(header => {
-      if (subheaders.hasOwnProperty(header.key)) {
-        return subheaders[header.key].map(subheader => subheader.text)
-      }
-      return header.text
-    })
-
-    return [tsvHeaders, tsvData]
-  }
-
   const render = (tableData: TableItem[]) => {
-    console.log(tableData)
-    const [tsvHeaders, tsvData] = tsvDataMap(tableData)
-    const separator = "\t"
+    console.log(tableData);
```

Comment:
rogue console.log 😆 !

Reply:
odd that linting didn't catch this

Reply:
Ooh didn't notice that lol Adding that in this [PR](https://github.com/chanzuckerberg/aspen/compare/thuang-eslint-no-console?expand=1) PTAL thank you!